package.json

```diff
@@ -77,7 +77,7 @@
 		"eslint-plugin-jsx-a11y": "6.0.2",
 		"eslint-plugin-react": "7.5.1",
 		"expose-loader": "0.7.3",
-		"extract-text-webpack-plugin": "3.0.0",
+		"extract-text-webpack-plugin": "4.0.0-alpha.0",
```

Comment:
https://github.com/webpack-contrib/extract-text-webpack-plugin/releases/tag/v4.0.0-beta.0 - beta is out, but still not a stable one. We probably should proceed anyway.